libr/debug/p/debug_native.c

```diff
@@ -884,7 +884,10 @@ static int r_debug_native_kill (RDebug *dbg, int pid, int tid, int sig) {
 	int ret = false;
 	if (pid == 0) pid = dbg->pid;
 #if __WINDOWS__ && !__CYGWIN__
-	ret = w32_terminate_process (dbg, pid);
+	if (sig==0)
+		ret = true;
+	else
+		ret = w32_terminate_process (dbg, pid);
```

Comment:
what about `ret = (sig > 0)? w32_terminate_process (dbg, pid): true;` ?

iirc, this way r2 will never know if the target process is dead, because it will always return true when sig==0, right?